spec/support/feature_helpers.rb

```diff
@@ -115,6 +115,11 @@ def start_early_career_payments_claim
     Claim.by_policy(EarlyCareerPayments).order(:created_at).last
   end
 
+  def start_levelling_up_premium_payments_claim
```

Comment:
As mentioned earlier, this maybe a combined journey start vs start_(ecp|lup)_claim